contribs/application/src/main/java/org/matsim/application/prepare/population/ResolveGridCoordinates.java

```diff
@@ -106,6 +111,12 @@ public Integer call() throws Exception {
 
 						if (network != null) {
 							Link link = NetworkUtils.getNearestLink(network, act.getCoord());
+							if(link.getAllowedModes().contains(TransportMode.car)){
```

Comment:
I think you wanted to check if a link does con contain car ?

Reply:
correct. thanks for catching the missing `!`. However, didn't we agree on enforcing to use a car network more strongly? Should I change the warn message to an exception? IllegalArgumentException?